pennylane/ops/cv.py

```diff
@@ -1164,7 +1165,40 @@ def _heisenberg_rep(p):
         return np.array([0, 1, 0])
 
 
-class P(CVObservable):
+class X(QuadX):
+    r"""
+    This class is deprecated and will be removed in PennyLane v0.33.
+    Please use qml.QuadX instead.
```

Comment:
Can we update this to be a warning, to be just above `Details`, and to link dynamically to `qml.QuadX`. The same applies to `QuadP`.